polling_stations/templates/postcode_view.html

```diff
@@ -41,25 +41,53 @@
     {% if error %}
 
       {% if error == 'Mapit error 403: Rate limit exceeded' %}
-      {% blocktrans %}
-      <h2>Sorry, we're a bit busy right now!</h2>
-      <p>Try again in a few minutes.</p>
-      {% endblocktrans %}
-      {% else %}
-      {% blocktrans %}
-      <h2>Sorry, we can't find {{ postcode }}</h2>
-      <p>This doesn't appear to be a valid postcode.</p>
-      {% endblocktrans %}
-      <form method="post" action="{% url 'home' %}" class="form form-inline">
-          {% csrf_token %}
-          <label class="card_header">{% trans "Enter Your Postcode" %}
-            <input id="id_postcode" name="postcode" type="text" aria-describedby="postcode_help_text" autofocus >
-          </label>
-          <p class="help-text" id="postcode_help_text">
-            {% trans 'e.g.' %} <a href="{{ example_postcode.url }}">{{ example_postcode.display }}</a>
+
+        {% blocktrans %}
+        <h2>Sorry, we're a bit busy right now!</h2>
+        <p>Try again in a few minutes.</p>
+        {% endblocktrans %}
+
+      {% elif error == 'There are no upcoming elections in your area' %}
+
+        <h2>{{ error }}</h2>
+        {% if council.phone %}
+          {% blocktrans with council.name as council_name and council.phone as council_phone %}
+          <p>
+            We're not aware of any upcoming elections in your area. If you think there may be elections in your area, please call your council to check and then <a href="https://democracyclub.org.uk/contact/">let us know</a>.*
+            You can contact {{ council_name }} on <strong><a href="tel:{{ council_phone }}">{{ council_phone }}</a></strong>
           </p>
-          <button type="submit" class="button" id="submit-postcode">{% trans "Find your Polling Station" %}</button>
-      </form>
+          {% endblocktrans %}
+        {% else %}
+          {% blocktrans with council.name as council_name %}
+          <p>
+            We're not aware of any upcoming elections in your area. If you think there may be elections in your area, please call {{ council_name }} to check and then <a href="https://democracyclub.org.uk/contact/">let us know</a>.*
+          </p>
+          {% endblocktrans %}
+        {% endif %}
+        <p>
+          <small>
+            * Why are we not certain? Because there is no canonical source of UK elections.
```

Comment:
I asked some people from Bournemouth Council about this (they happened to email, so I showed them the wording we wanted to use). They said:

> Two colleagues of mine did comment on the use of the word 'canonical' though, not knowing what it meant and thinking electors might not either.  We thought something like 'definitive' might be more likely to be understood?

@JoeMitchell do you have reckons?